calico/_includes/content/calico-windows-install.md

```diff
@@ -0,0 +1,114 @@
+{%- if include.networkingType == "vxlan" %}
+1. Ensure that BGP is disabled since you're using VXLAN.
```

Comment:
Suggest this...


1. Ensure that BGP is disabled.  
  -  If you installed {{site.prodname}} using the {% include open-new-window.html text='manifest' url='[URL](https://projectcalico.docs.tigera.io/manifests/calico-vxlan.yaml)' %}, BGP is already disabled.
  - If you installed {{site.prodname}} using the operator, run this command:

   ```bash
   kubectl patch installation default --type=merge -p '{"spec": {"calicoNetwork": {"bgp": "Disabled"}}}'
   ```